index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Portfolio | Edwin Gichuhi</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header class="header">
+            <nav class="nav">
+                <a href="#">My Logo</a>
+                <span><img src="./assets/icons/Union.png" alt="menu"></span>
+            </nav>
+        </header>
+        <section class="headline-section">
+            <p class="intro-text">I'm Edwin, Glad to see you!</p>
```

Comment:
- [ ] it would be better if you use the `<h1>` tag cause the main header of the portfolio will improve the readability, user experience, and SEO , it would be better if you keep only one h1 on a page for performance reasons

Reply:
Now I see the importance of the <h1> tag. Thanks for the tip

Reply:
Fixed.